src/services/baseService.ts

```diff
@@ -38,6 +41,9 @@ export abstract class BaseService {
     this.resourceGroup = this.configService.getResourceGroupName();
     this.deploymentName = this.configService.getDeploymentName();
     this.artifactName = this.configService.getArtifactName(this.deploymentName);
+    this.runtime = this.configService.getRuntime();
+    this.pythonTarget = this.runtime.language === SupportedRuntimeLanguage.PYTHON;
```

Comment:
Similar to above - doesn't feel like python/linux related variables should exist on the base service.  Also, since they are boolean variables i would rename to something like `isPythonTarget` && `isLinuxTarget`